src/ripple/net/impl/RPCCall.cpp

```diff
@@ -144,6 +145,21 @@ class RPCParser
         }
     }
 
+    static Json::Value validPublicKey (std::string const& strPk)
```

Comment:
This function should return a `bool` and not a `Json::Value`.

Reply:
Done